tests/cli/test_cli.py

```diff
@@ -376,6 +377,7 @@ def __enter__(self):
         temp_worksheet = _run_command([cl, 'new', random_name()])
         self.worksheets.append(temp_worksheet)
         _run_command([cl, 'work', temp_worksheet])
+        self.disk_quota = _run_command([cl, 'uinfo', 'codalab', '-f', 'disk']).split(' ')[2]
```

Comment:
We can omit "codalab" to get the current user's info
```suggestion
        self.disk_quota = _run_command([cl, 'uinfo', '-f', 'disk']).split(' ')[2]
```